python/graphstorm/gsf.py

```diff
@@ -54,7 +56,29 @@
                                  LinkPredictWeightedDistMultDecoder)
 from .tracker import get_task_tracker_class
 
-def initialize(ip_config, backend):
+def init_wholegraph():
+    """ Initialize Wholegraph"""
+    import pylibwholegraph.torch as wgth
+    import pylibwholegraph.binding.wholememory_binding as wmb
+
+    class Options: # pylint: disable=missing-class-docstring
+        pass
+    Options.launch_agent = 'pytorch'
+    Options.launch_env_name_world_rank = 'RANK'
```

Comment:
Can you check whether os.environ has these keywords?

Reply:
It does I think but we pass `Options` when calling `distributed_launch`.So We need to initialize `Options` class anyway.